<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="Rohit" name="com.testCases.Test2" tests="10" failures="2" timestamp="30 Oct 2021 12:09:53 GMT" time="18.288" errors="0">
  <testcase name="Dashboard" time="0.631" classname="com.testCases.Test2">
    <skipped/>
  </testcase> <!-- Dashboard -->
  <testcase name="Dashboard" time="1.386" classname="com.testCases.Test2">
    <failure type="java.lang.AssertionError" message="expected:&lt;JavaByKiran | Dashboard&gt; but was:&lt;JavaByKiran | Dashboard1&gt;">
      <![CDATA[java.lang.AssertionError: expected:<JavaByKiran | Dashboard> but was:<JavaByKiran | Dashboard1>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
at com.testCases.Test2.Dashboard(Test2.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- Dashboard -->
  <testcase name="Dashboard" time="0.573" classname="com.testCases.Test2">
    <failure type="java.lang.AssertionError" message="expected:&lt;JavaByKiran | Dashboard&gt; but was:&lt;JavaByKiran | Dashboard1&gt;">
      <![CDATA[java.lang.AssertionError: expected:<JavaByKiran | Dashboard> but was:<JavaByKiran | Dashboard1>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
at com.testCases.Test2.Dashboard(Test2.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- Dashboard -->
  <testcase name="loginTest" time="0.325" classname="com.testCases.Test2"/>
  <testcase name="listOfCourses" time="1.109" classname="com.testCases.Test2"/>
  <testcase name="listOfCourses" time="1.681" classname="com.testCases.Test2"/>
  <testcase name="DashboardUserText" time="5.015" classname="com.testCases.Test2"/>
  <testcase name="Dashboard" time="0.646" classname="com.testCases.Test2">
    <skipped/>
  </testcase> <!-- Dashboard -->
  <testcase name="DashboardUserText" time="6.492" classname="com.testCases.Test2"/>
  <testcase name="loginTest" time="0.430" classname="com.testCases.Test2"/>
</testsuite> <!-- com.testCases.Test2 -->
